packages/cannon-worker/CHANGELOG.md

```diff
@@ -0,0 +1,159 @@
+# @react-three/cannon Changelog
```

Comment:
This file belongs in `packages/react-three-cannon`

Reply:
I just saw it further down, no need to duplicate the contents of this file.

Reply:
```suggestion
# @pmndrs/cannon-worker Changelog
```

Reply:
My bad :) left over from a copy-paste. Fixed.